index.js

```diff
@@ -0,0 +1,163 @@
+import express from "express";
+import cors from "cors";
+import climbingRoutes from "./data/climbing-routes.json" assert { type: "json" };
+import listEndpoints from "express-list-endpoints";
+
+const port = process.env.PORT || 9000;
+const app = express();
+
+app.use(cors());
+app.use(express.json());
+
+// API Documentation with express-list-endpoints
+app.get("/", (_, res) => {
+  res.json({
+    message:
+      "Welcome to the Climbing Routes API! Here are the available endpoints:",
+    endpoints: listEndpoints(app),
+  });
+});
+
+// Return all climbing routes
+// http://localhost:9000/climbingRoutes
+app.get("/climbingRoutes", (_, res) => {
+  res.json(climbingRoutes);
+});
```

Comment:
Instead of creating different endpoints for each filter/sorting option, you should just include all of the query params under this endpoint. This way you can also chain the filters, if anyone wants to filter on e.g. type _and_ difficulty.

This is good practice to make your API RESTful.